packages/discovery-provider/ddl/migrations/0095_shorten_challenge_ids.sql

```diff
@@ -0,0 +1,863 @@
+--
```

Comment:
maybe an issue this isn't all in one transaction?

Reply:
yeah good point!